### DISCUSSION TITLE
Pimcode admin Collections issue

### DISCUSSION THREAD

**Discussion by abrar123alhaq** [pimcore/pimcore#13665] [Q&A / Support]
Hello,

When I open collection from pimcore, then close collection and try to reopen it is not opening 

![image](https://user-images.githubusercontent.com/832552/203567931-b861eb80-c8d0-43dd-b5cd-22456b5b7876.png)

![image](https://user-images.githubusercontent.com/832552/203567704-c5a2be60-435a-4ce9-b35c-0a0dc2c265ef.png)

Please suggest some solution.

**Comment by fashxp** [comment]
it is a bug. created a ticket in the corresponding repository. if you don't have access, please contact [enterprise-support@pimcore.com](mailto:enterprise-support@pimcore.com)

**Comment by fashxp** [comment]
for reference: https://github.com/pimcore/portal-engine/issues/286
